game_eggs/steamcmd_servers/ground_branch/egg-ground-branch.json

```diff
@@ -0,0 +1,132 @@
+            "env_variable": "MAX_PLAYERS",
+            "default_value": "16",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer|min:1",
+            "field_type": "text"
+        },
+        {
+            "name": "[Repair] Validate Server Files",
+            "description": "Leave empty (no value) for OFF or type \"true\" or \"1\" for ON. Validates all server files when Automatic Updates is enabled. Note: This will significantly increase server startup times, so it is recommended to only enable this when needed.",
+            "env_variable": "VALIDATE",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "string|nullable",
+            "field_type": "text"
+        },
+        {
+            "name": "[System] Ground Branch Dedicated Server App ID",
+            "description": "Steam App ID used for installation and updates. Cannot be changed.",
+            "env_variable": "SRCDS_APPID",
+            "default_value": "476400",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|in:476400",
+            "field_type": "text"
+        },
+        {
+            "name": "[System] Use Windows Branch",
+            "description": "Tells the installer\/updater to only download the Windows branch of the server (the only branch currently available) so that it can run on Wine. Cannot be changed.",
+            "env_variable": "WINDOWS_INSTALL",
+            "default_value": "1",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|in:1",
+            "field_type": "text"
+        },
+        {
+            "name": "[System] WINEDEBUG",
+            "description": "Used to suppress WINE FIXME messages. Rarely needs to be changed.",
+            "env_variable": "WINEDEBUG",
+            "default_value": "-all",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "string|nullable",
+            "field_type": "text"
+        },
+        {
+            "name": "[System] WINEARCH",
+            "description": "Used for compatibility. Cannot be changed.",
+            "env_variable": "WINEARCH",
+            "default_value": "win64",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|in:win64",
+            "field_type": "text"
+        },
+        {
+            "name": "[System] WINEPATH",
+            "description": "Used for compatibility. Cannot be changed.",
+            "env_variable": "WINEPATH",
+            "default_value": "\/home\/container",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|in:\/home\/container",
+            "field_type": "text"
+        }
```

Comment:
does it preform any beter or spit out less errors when the winetrick dotnet46, vcrun2015, 2017, 2019 is installed? ( to stop hiding the errors set WINEDEBUG to `+all` or just empty)
![image](https://github.com/parkervcp/eggs/assets/67589015/d0b1e79c-6763-4568-8769-4d2c5e02c11f)